fluent-plugin-logdna.gemspec

```diff
@@ -3,7 +3,7 @@ require 'date'
 
 Gem::Specification.new do |s|
   s.name        = 'fluent-plugin-logdna'
-  s.version     = '0.1.7'
+  s.version     = '0.1.8'
```

Comment:
i think we want to do 0.1.7.1? i'm worried that this packaging will conflict with whenever upstream releases 0.1.8